app/utils/code-mirror-documents.ts

```diff
@@ -1,27 +1,33 @@
 import { tracked } from '@glimmer/tracking';
 import parseDiffAsDocument from 'codecrafters-frontend/utils/parse-diff-as-document';
 
+export type LinesRange = { startLine: number; endLine: number };
```

Comment:
```suggestion
export type LineRange = { startLine: number; endLine: number };
```

(the plural for this would be cleaner, `lineRanges` vs. `linesRanges`, also similar usage: [link](https://github.com/codemirror/comment/blob/269b9af21d68b1a71fa40bbf6c1e41c665e6e7d0/src/comment.ts#L122))

Reply:
Alright, renamed to `LineRange`, thanks for the tip! Also moved from `utils/code-mirror-documents` into `components/code-mirror`, not sure why I didn't put it there in the first place. Merging once tests pass.